fossr/README.md

```diff
@@ -0,0 +1,16 @@
+# FOSSR
+FOSSR aims to create an Open Science Cloud for data integration and promotes sharing and access to research results.
+FOSSR ? ?Fostering Open Science in Social Science Research? is a project financed through the funds of the National Recovery and Resilience Plan (PNRR) ? NextGenerationEU under Grant agreement MUR IR00008.
```

Comment:
It appears that a few characters may not have survived some copy-paste operation(s), and got turned to `?` in what I see.

* `FOSSR ? ?Fostering`
* `Research? is`
* `(PNRR) ? NextGenerationEU`